### ISSUE TITLE
when useWindow=false, resize event is ignored

### ISSUE THREAD

**Issue by realmika** [CassetteRocks/react-infinite-scroller#169]
**Describe the bug**
I'm running infinite scroller within a div. When I resize the window to make it bigger, I can see white space at the end and the loadMore function is never called.

**To Reproduce**
https://codesandbox.io/s/ojo9vlvp35

Start the page with a very low window height, then when it has finished loading increase the height of the window and see that white space is showing and more items are not loaded.

**Expected behavior**
Items to be loaded when the parentNode has changed size.

**Screenshots**
![image](https://user-images.githubusercontent.com/1054229/43148174-0a467d26-8f65-11e8-847b-06a06de56ac7.png)

**Device (please complete the following information):**
ubuntu 18.04, chrome 67

**Comment by azu-b** [comment]
I tried to reproduce the bug in a macOS High Sierra 10.13.6 and Chrome 68, and I couldn't 🤔 
[Record It](http://g.recordit.co/dgxQi0ahiE.gif).

If it still happens to you, could you please upload a gif screenshot to see exactly how to reproduce the bug?

Thanks 😄

**Comment by realmika** [comment]
Hi there,

I believe an important step is not to scroll before resizing the window. Here you go, it's not a GIF but should be viewable in any browser: [not-loading.ogv](https://www.dropbox.com/s/vh6gr7b7k5rt4m7/not-loading.ogv?dl=0)

It still happens to me. Let's see if maybe it's specific to Chrome on Linux?

**Comment by azu-b** [comment]
Hello again! Although I can't upload my gifs of me reproducing the bug, now I can tell the bug happens also in Chrome and in Safari in Mac 😢 .

**Comment by realmika** [comment]
Ah perfect, one step closer to understanding the issue :)

**Comment by danbovey** [comment]
Thanks for the reproductions, the no-loading.ogv video makes it clear that we currently don't support `useWindow` without the scroll container being a fixed height. Looks like the container you have in the demo is styled like `height: 90vh`.

**Comment by realmika** [comment]
Yes, this is true. It needs to be dynamic for my project. Anyone up for fixing this? I'm pressed for time, but if no one else does it I might try to give it a shot in the weekend

**Comment by Alphonse-Indpro** [comment]
Any solution to this? Many pull request related is not yet merged. Is this repository is not maintained?

**Comment by mikabytes** [comment]
@Alphonse-Indpro Ah two years later. I never got around to fix it. I migrated to Web Components and stopped using React altogether.
